src/test/regress/specs/isolation_data_migration.spec

```diff
@@ -0,0 +1,73 @@
+setup
+{
+    CREATE TABLE migration_table (test_id integer NOT NULL, data text);
+}
+
+teardown
+{
+    DROP TABLE migration_table;
+}
+
+session "s1"
+
+step "s1-begin"
+{
+    BEGIN;
+}
+
+step "s1-begin-serializable"
+{
+	BEGIN TRANSACTION ISOLATION LEVEL SERIALIZABLE;
+	SELECT 1;
+}
+
+step "s1-create_distributed_table"
+{
+    SELECT create_distributed_table('migration_table', 'test_id');
+}
+
+step "s1-commit"
+{
+    COMMIT;
+}
+
+session "s2"
+
+step "s2-begin"
+{
+    BEGIN;
+}
+
+step "s2-copy"
+{
+    COPY migration_table FROM PROGRAM 'echo 1,hello' WITH CSV;
+}
+
+step "s2-insert"
+{
+    INSERT INTO migration_table VALUES (1, 'hello');
+}
+
+step "s2-commit"
+{
+    COMMIT;
+}
+
+step "s2-select"
+{
+	SELECT * FROM migration_table;
```

Comment:
Realize this has only a single row for now, but it seems better to start with an ORDER BY here.